target.cpp

```diff
@@ -336,8 +336,10 @@ void TargetClient::OnPaint(wxPaintEvent& WXUNUSED(evt))
     }
 
     // Draw labels
-    dc.DrawText(_("RA"), leftEdge, center.y - 15);
-    dc.DrawText(_("Dec"), center.x - 35, topEdge - 3);
+    int d_width = 10, d_height = 10;

```

Comment:
minor, these initializers are not needed, let's remove them to avoid confusion
```suggestion
    int d_width, d_height;
```

Reply:
curious, what does the leading `d_` prefix stand for in these variable names?

Reply:
Agreed. It took me a minute to recall what I meant by d_ - probably a shortcut for 'digit'.